test/e2e/serviceloadbalancers.go

```diff
@@ -203,7 +203,9 @@ func (s *ingManager) test(path string) error {
 	})
 }
 
-var _ = Describe("ServiceLoadBalancer", func() {
+// TODO(ihmccreery) Skipped originally in #14988, never sent follow-up PR, as
+// far as I can tell.
+var _ = Describe("ServiceLoadBalancer [Skipped]", func() {
```

Comment:
this will make this skipped by default in non-Jenkins runs, too - is this what we want?